js/gdm/authPrompt.js

```diff
@@ -600,18 +611,35 @@ const AuthPrompt = new Lang.Class({
         this.emit('cancelled');
     },
 
+    _loadCustomerSupportFile: function() {
+        if (this._customerSupportKeyFile)
+            return this._customerSupportKeyFile;
+
+        for (let path of CUSTOMER_SUPPORT_LOCATIONS) {
+            let keyFile = new GLib.KeyFile();
+            try {
+                keyFile.load_from_file(path, GLib.KeyFileFlags.NONE);
+                this._csutomerSupportKeyFile = keyFile;
```

Comment:
`_csutomerSupportKeyFile` -> `_customerSupportKeyFile`. Otherwise the early return above these lines will never be useful